aarch64/src/l.S

```diff
@@ -2,15 +2,82 @@
 
 STACKSZ = 4096*4
 
+.equ	CURRENTEL_EL,		(1<<3) | (1<<2)
```

Comment:
Don't use `.equ`; just use `=` for consistency with the rest of the code.

Reply:
where are we using `=` in assembly?  I see `.set`, which is apparently mutable, so felt `.equ` would be better.

Reply:
`=` seemed briefer and serves largely the same purpose.